### PR TITLE
Prettier: Fix all markdown files and add markdown files to CI verify step

### PR DESCRIPTION
**Pull request by torkelo**
This seems to have been missed when we enabled prettier for markdown files.

### CODE REVIEW

**Review by hugohaggmark — APPROVED** [review]
LGTM! 🎉

File: CHANGELOG.md [hunk 1]
```diff
@@ -291,9 +287,9 @@ Panel queries and/or annotation queries that used more than one statistic will b
...
 - **InfluxDB:** InfluxQL: make measurement search case insensitive. [#34563](https://github.com/grafana/grafana/pull/34563), [@gabor](https://github.com/gabor)
 - **Legacy Alerting:** Replace simplejson with a struct in webhook notification channel. [#34952](https://github.com/grafana/grafana/pull/34952), [@KEVISONG](https://github.com/KEVISONG)
-- **Legend:** Updates display name for Last (not null) to just Last*. [#35633](https://github.com/grafana/grafana/pull/35633), [@torkelo](https://github.com/torkelo)
+- **Legend:** Updates display name for Last (not null) to just Last\*. [#35633](https://github.com/grafana/grafana/pull/35633), [@torkelo](https://github.com/torkelo)
 - **Logs panel:** Add option to show common labels. [#36166](https://github.com/grafana/grafana/pull/36166), [@ivanahuckova](https://github.com/ivanahuckova)
-- **Loki:** Add $__range variable. [#36175](https://github.com/grafana/grafana/pull/36175), [@ivanahuckova](https://github.com/ivanahuckova)
+- **Loki:** Add $\_\_range variable. [#36175](https://github.com/grafana/grafana/pull/36175), [@ivanahuckova](https://github.com/ivanahuckova)
```
Comment (Clarity-89) [inline CHANGELOG.md:292]: Wonder if this character escape actually supposed to be here?

Comment (torkelo) [reply CHANGELOG.md:292]: it seems to work the same at least.

Comment (torkelo) [reply CHANGELOG.md:292]: https://github.com/prettier/prettier/issues/3836

Comment (torkelo) [reply CHANGELOG.md:292]: looks like there is no way to disable this

**Pull request merged**